stac_fastapi/types/stac_fastapi/types/requests.py

```diff
@@ -1,10 +1,15 @@
 """Requests helpers."""
 
+import os
+
 from starlette.requests import Request
 
 
 def get_base_url(request: Request) -> str:
     """Get base URL with respect of APIRouter prefix."""
+    if base_url := os.environ.get("REQUEST_BASE_URL"):
+        return base_url
+
     app = request.app
     if not app.state.router_prefix:
```

Comment:
Was there a reason to _not_ use the same mechanism as `router_prefix`, i.e. add a new `base_url: Optional[str]` attribute to the app state?